test/IDE/complete_global_actorisolation.swift

```diff
@@ -0,0 +1,178 @@
+// REQUIRES: concurrency
+// RUN: %target-swift-ide-test -batch-code-completion -source-filename %s -filecheck %raw-FileCheck -completion-output-dir %t -enable-experimental-concurrency
+
+class MyNonSendable {}
+struct MySendable {}
+
+actor MyActor {}
+
+@globalActor
+actor MyGlobalActor {
+  static var shared = MyActor()
```

Comment:
I believe this must be `MyGlobalActor` in the new syntax.

> The type of `shared` must be of the enclosing actor type

Reply:
Sorry, I missed that part. Fixed it now.